app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class TasksController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_task, only: [:show, :edit, :destroy, :update]
+
+  def index
+    @tasks = current_user.tasks.all
+  end
+
+  def new
+    @task = current_user.tasks.new
+  end
+
+  def create
+    @task = current_user.tasks.create(task_params)
+    if @task.save
+      redirect_to [current_user, @task], notice: "Task has successfully created."
+    else
+      redirect_to new_user_task_path, alert: @task.errors.full_messages.to_sentence
```

Comment:
check render

Reply:
Ammar bhae says not use render. Just use redirect_to